tools/crm_mon.c

```diff
@@ -2009,10 +2012,10 @@ print_node_attribute(gpointer name, gpointer user_data)
     value = pe_node_attribute_raw(data->node, name);
 
     /* Print attribute name and value */
-    switch (output_format) {
+    switch (data->fmt) {
         case mon_output_plain:
         case mon_output_console:
-            print_as("    + %-32s\t: %-10s", (char *)name, value);
+            print_as(output_format, "    + %-32s\t: %-10s", (char *)name, value);
```

Comment:
Do you mean data->fmt here and below?

Reply:
Yes and in fact I fix this in a later patch, but it's worth fixing in the earlier patch.